### ISSUE TITLE
Loop sound track

### ISSUE THREAD

**Issue by shadowvzs** [Return-To-The-Roots/s25client#426]
can make loop button for 1 music only?

**Comment by Spikeone** [comment]
I don't think we need such a button as you can simply play that one music title in any media player.
Although this could be done if we'd add some kind of playlists but... I also see no use here.

Any ideas @Flamefire ?

**Comment by Flamefire** [comment]
Already exists: We do have the music player in which you define a playlist. You can add a song there and set the repeat count up. I proposed to remove the repeat count and simply repeat the playlist forever (see #894)
